server/models/meal.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const mealSchema = new Schema({
+  title: { type: String, required: true },
+  picURL: String,
+  price: { type: Number, required: true },
+  // ie: 2 people, 4 people
+  servingSize: { type: String, required: true },
+  // ie: family style, buffet
+  servingType: String,
+  cuisineType: [{ type: String, required: true }],
+  ingredients: { type: String, required: true },
+  createdAt: { type: Date, default: Date.now },
```

Comment:
Your diagram also includes `requirements`, which I think would be good to include

Reply:
Good catch.  I missed adding that.